atlas/templates/kind/.id.gotmpl

```diff
@@ -0,0 +1 @@
+{{ .Helm.GetName }}
```

Comment:
This file shouldn't be included. It's intended as a per developer identifier for determining namespace to install to.

Reply:
We also need to update https://github.com/infobloxopen/atlas-cli/blob/master/atlas/templates/.gitignore.gotmpl to ignore file named `/.id`

Reply:
Yeah, user can remove this file and generate new one based on his git config, however I assume the service name as a "default" namespace also is suitable. Considering that, should we remove this file anyway?